charts/app/values.yaml

```diff
@@ -0,0 +1,3 @@
+annotations:
+  spoc: "Niclas Schumacher nsch@nuuday.dk"
+  application_service: "saga"
```

Comment:
we use saga-dev in azure, would make sense to match the same convention. 
Thoughts? 

If yes, add the variables to the helper chart methods which then automatically will the environment.

Reply:
good point.